src/main/scala/com/fulcrumgenomics/umi/VanillaUmiConsensusCaller.scala

```diff
@@ -60,19 +68,69 @@ case class VanillaUmiConsensusCallerOptions
   minInputBaseQuality: PhredScore         = DefaultMinInputBaseQuality,
   minConsensusBaseQuality: PhredScore     = DefaultMinConsensusBaseQuality,
   minReads: Int                           = DefaultMinReads,
-  requireConsensusForBothPairs: Boolean   = DefaultRequireConsensusForBothPairs
+  requireConsensusForBothPairs: Boolean   = DefaultRequireConsensusForBothPairs,
+  producePerBaseTags: Boolean             = DefaultProducePerBaseTags
 )
 
-/** Stores information about a read to be fed into a consensus. */
+/**
+  * Object that encapsulates the various consensus related tags that are added to consensus reads
+  * at both the per-read and per-base level.
+  *
+  * Currently only contains tags for single-strand consensus reads, but with a view to using the following
+  * names for consistency if/when we add duplex calling:
+  *     Value                 AB  BA  Final
+  *     ===================== ==  ==  =====
+  *     bases                 ac  bc  bases
+  *     quals                 aq  bq  quals
+  *     per-read-depth        aD  bD  cD
+  *     per-read-min-depth    aM  bM  cM
+  *     per-read-error-count  aE  bE  cE
+  *     per-base-depth        ad  bd  cd
+  *     per-base-error-count  ae  be  ce
+  */
+object ConsensusTags {
+  object PerBase {
+    /** The per-base number of raw-reads contributing to the consensus (stored as a short[]). */
+    val RawReadCount    = "cd" // consensus depth
+    /** The number of bases at each position that disagreed with the final consensus call (stored as a short[]). */
+    val RawReadErrors   = "ce" // consensus errors
+  }
+
+  object PerRead {
+    /** The number of raw reads that contributed to the consensus. I.e. the maximum of the per-base raw-read counts. */
+    val RawReadCount    = "cD" // consensus Depth
```

Comment:
Likely noone but us will be looking at the long names for the tags - so I don't know that it matters here so much as wherever we explain in more detail.